Utilities/Doxygen/DoxygenConfig.cmake

```diff
@@ -164,6 +164,7 @@ set(DOXYGEN_GRAPHICAL_HIERARCHY "NO")
 set(DOXYGEN_DOT_IMAGE_FORMAT "svg")
 set(DOXYGEN_DOT_GRAPH_MAX_NODES "150")
 set(DOXYGEN_DOT_MULTI_TARGETS "YES")
+set(DOXYGEN_CITE_BIB_FILES "${ITK_SOURCE_DIR}/Documentation/Doxygen/doxygen.bib")
```

Comment:
This line should probably be added to https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/DoxygenConfig.cmake too.